docs/operate/reference/viam-server/_index.md

```diff
@@ -107,8 +107,8 @@ maintenance : {
 Log messages appear under the [**LOGS** tab](/manage/troubleshoot/troubleshoot/#check-logs) for a machine.
 
 The default log level for `viam-server` and any running resources is `"Info"`.
-
-You can set log levels for individual components or other resources by adding the `log_configuration` option to the resources' JSON configuration:
+If you need more logs for individual resource, click **Enable debug logs** in the **...** menu on the resource.
+To set other log levels for individual resources, add the `log_configuration` option to the resources' JSON configuration:
 
 ```json
 "log_configuration": {
```

Comment:
[opt] There was a follow-up question about what level this goes at. Might be good to put this example into a larger JSON context (though bot did answer correctly I think?)

Reply:
It does have hopefully enough context with the attributes field next to it